README.md

```diff
@@ -192,126 +174,22 @@ trainer = Trainer(logger=wandb_logger)
 
-**Explore reports in [The Gallery →](https://app.wandb.ai/gallery) | Read the [Docs](https://docs.wandb.com/reports)**
+&nbsp;
 
-Once you have experiments in W&B, you can visualize and document results in Reports with just a few clicks. Here's a quick [demo video](https://www.youtube.com/watch?v=jWBGKGAjt6w&t=2s&ab_channel=Weights%26Biases).
 
-![](https://i.imgur.com/dn0Dyd8.png)
 
-# 🏺 Version control datasets and models with Artifacts
-Git and GitHub make code version control easy,
-but they're not optimized for tracking the other parts of the ML pipeline:
-datasets, models, and other large binary files.
-
-W&B's Artifacts are.
-With just a few extra lines of code,
-you can start tracking you and your team's outputs,
-all directly linked to run.
-
-### [Try Artifacts in a Colab →](http://wandb.me/artifacts-colab)
-
-![](https://i.imgur.com/zvBWhGx.png)
-
-### Common use cases
-- **Pipeline Management:** Track and visualize the inputs and outputs of your runs as a graph
-- **Don't Repeat Yourself™:** Prevent the duplication of compute effort
-- **Sharing Data in Teams:** Collaborate on models and datasets without all the headaches
-
-![](https://i.imgur.com/w92cYQm.png)
-
-**Learn about Artifacts [here →](https://www.wandb.com/articles/announcing-artifacts) | Read the [Docs](https://docs.wandb.com/artifacts)**
-
-# 💻  Run W&B Server Locally
-W&amp;B Server is a self-hosted Weights &amp; Biases server.  Securely and quickly deploy a W&amp;B production server in Docker, Kubernetes, or in a privately-managed cloud.  Learn more about setting up a [production W&amp;B deployment →](https://docs.wandb.ai/guides/self-hosted/setup).
-
-## Quickstart
-1. On a machine with [Docker](https://docker.com) and [Python](https://www.python.org/) installed, run:
-    ```
-    1 pip install wandb --upgrade
-    2 wandb server start
-    ```
-2. Generate a free license from the [Deployer](https://deploy.wandb.ai/).
-3. Add it to your W&amp;B Server's localhost's settings.
-
-  **Paste the license in the /system-admin page on your localhost**
-
-  ![2022-02-24 22 13 59](https://user-images.githubusercontent.com/25806817/166265834-6a9d1be8-2af5-4c63-872e-8e5b3e4082aa.gif)
-
-## Docker
-Running `wandb server start` will start our server and forward port 8080 on the host.  To have other machines report metrics to this server run: `wandb login --host=http://X.X.X.X:8080`.
-
-Use Docker to manually run W&amp;B Local:
-```
-docker run --rm -d -v wandb:/vol -p 8080:8080 --name wandb-local wandb/local
-```
+# Contact
+If you have any questions, please don't hesitate to ask in our [user forum](http://wandb.me/forum). For wandb bugs and feature requests please visit [GitHub Issues](https://github.com/wandb/wandb/issues). For professional support please Contact Us. Be a part of the W&B Community, post your qeustions at [W&B Community](https://community.wandb.ai/). Stay connected with the latest updates with [W&B Fully Connected](https://wandb.ai/fully-connected).
```

Comment:
```suggestion
If you have any questions, please don't hesitate to ask in our [user forum](http://wandb.me/forum). For wandb bugs and feature requests please visit [GitHub Issues](https://github.com/wandb/wandb/issues). For professional support please Contact Us. Be a part of the W&B Community, post your questions at [W&B Community](https://community.wandb.ai/). Stay connected with the latest updates with [W&B Fully Connected](https://wandb.ai/fully-connected).
```

Reply:
Contact Us goes to https://wandb.ai/site/contact?

Reply:
Oops. I'll add this in the next PR.